src/utils/tezos/estimate.ts

```diff
@@ -1,25 +1,40 @@
 import BigNumber from "bignumber.js";
 
-import { addressExists, makeToolkit, operationsToBatchParams, sumTez } from "./helpers";
-import { AccountOperations } from "../../types/AccountOperations";
+import { addressExists, makeToolkit, operationsToBatchParams } from "./helpers";
+import { AccountOperations, EstimatedAccountOperations } from "../../types/AccountOperations";
 import { Network } from "../../types/Network";
 
+export type Estimation = {
+  storageLimit: number;
+  gasLimit: number;
+  fee: number | BigNumber;
```

Comment:
since we are going to pass it into taquito I think it's better to be consistent with the type here and use `number`